variables.tf

```diff
@@ -142,3 +142,9 @@ EOF
     error_message = "The vpc_endpoints valid values are s3, sts and kinesis-streams."
   }
 }
+
+variable "token_lifetime_seconds" {
+  description = "(Optional) Token expiry lifetime. By default it is 2592000 (30 days)."
+  type = number
+  default = 2592000
```

Comment:
wouldn't be better to make it `-1` (no expiry) by default?

Reply:
I have used the same default Databricks currently uses in the underlying module.